src/pick/__init__.py

```diff
@@ -129,9 +148,16 @@ def draw(self, screen: "curses._CursesWindow") -> None:
         lines_to_draw = lines[scroll_top : scroll_top + max_rows]
 
         for line in lines_to_draw:
-            screen.addnstr(y, x, line, max_x - 2)
+            screen.addnstr(y, x, line, max_x // 2 - 2)
```

Comment:
If there is no descriptions at all, should we draw the line with half of the screen?

Reply:
I think we should not care too much about the performance here; iterating a list with at most hundreds of elements does not take too much resources. I think we can just caculate the `description_present` in this function.

Reply:
Is [87540ac](https://github.com/wong2/pick/pull/114/commits/87540acf464881cb6e06bdf81a24221fdcb7503c) good?